src/components/Join/Join.js

```diff
@@ -9,13 +9,13 @@ class Join extends Component {
     };
   }
 
-  handleChange = (e) => {
+  handleChange = (e) => { // eslint-disable-line
```

Comment:
In this way, we will always add `eslint-disable-line` to bypass this line.

If you don't like `.bind(this)`, I think we can move `() =>` into component like `onSubmit={() => this.handleSubmit()}`.